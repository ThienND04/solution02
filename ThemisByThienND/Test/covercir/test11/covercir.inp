4
61138 39142  43064
91513 58260  1755
65090 6206  58070
54917 29895  6122
