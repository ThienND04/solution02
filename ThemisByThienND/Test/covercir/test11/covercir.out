32529
49339
38984
28686
